packages/cozy-harvest-lib/src/components/cards/LaunchTriggerCard.jsx

```diff
@@ -68,7 +68,7 @@ export class LaunchTriggerCard extends PureComponent {
                     disabled={running}
                     onClick={launch}
                     subtle
-                    style="line-height:1.4"
+                    style={{ lineHeight: '1.4' }}
```

Comment:
Nit: It's weird to have this here. Styling the button should be done via its props, or utility classes, or if the style is really warranted, it should be extracted out of the render so that the same style object is used and the component not re-rendered.

I am OK for the fix but in those cases, I think adding a TODO here to say that the style should be extracted would be nice.

Reply:
Agree. We discuss this with @GoOz yesterday. For now there is no utility class allowing us to manage this case. However, I should have added added a TODO.